src/components/TransactionFlow/displays/SwapFromToDiagram.tsx

```diff
@@ -22,37 +22,37 @@ const Wrapper = styled.div`
   align-items: center;
   justify-content: center;
   margin: 18px 0;
+  background: #f8f8f8;
```

Comment:
use `COLORS`

Reply:
We don't have this color available. What would you prefer done?

Reply:
you can set a new key for this color :)

Reply:
Still missing key for this color